regression/scripts/battery/aircraft_discharge_comparisons.py

```diff
@@ -0,0 +1,568 @@
+# aircraft_discharge_comparisons.py
+# 
+# Created: Feb 2020, M. Clarke
+#          Sep 2020, M. Clarke 
+#          Jul 2021, R. Erhard
```

Comment:
Was she really the last one to edit this new file?

Reply:
no, will remove her name